public/js/library/new.js

```diff
@@ -1,122 +1,112 @@
-
-            document.getElementById("lngError").innerText = "";
-        } catch (e) {
-            document.getElementById("lngError").innerText = e;
-            lng.value = ""
-            errorList.innerHTML += `<li> ${e} </li>`;
-        }
-
-        try {
-            checkImageFileString(image.value, "Input Image");
-
-            document.getElementById("imageError").innerText = "";
-        } catch (e) {
-            document.getElementById("imageError").innerText = e;
-
-            errorList.innerHTML += `<li> ${e} </li>`;
-            image.value = null;
-        }
-
-        let genresForm, genres;
-        try {
-            genresForm = $("#genres input");
-            genres = genresForm.toArray();
-            genres = genres.filter((input) => input.checked);
-
-            document.getElementById("genresError").innerText = "";
-        } catch (e) {
-            if (typeof e === "string" && e.startsWith("VError")) {
-                e = e.substr(1);
-            }
-
-            document.getElementById("genresError").innerText = e;
-
-            errorList.innerHTML += `<li> ${e} </li>`;
-        }
-
-        try {
-            if (!isNumber(Number(fullnessVal))) throw "You must select a fullness!";
-
-            if (fullnessVal < 0 || fullnessVal > 5) throw "Improper Range on Fullness";
-
-            document.getElementById("fullnessError").innerText = "";
-        } catch (e) {
-            document.getElementById("fullnessError").innerText = e;
-            errorList.innerHTML += `<li> ${e} </li>`;
-        }
-
-        try {
-            if (genres.length === 0 && fullnessVal > 0) throw "You must select at least one genre if the library is non-empty!";
-            
-            if (genres.length > 0 && fullnessVal === 0) throw "You cannot select any genres if the library is empty!";
-
-            document.getElementById("genresError").innerText = "";
-        } catch (e) {
-            errorList.innerHTML += `<li> ${e} </li>`;
-            document.getElementById("genresError").innerText = "";
-        }
-        
-        if(errorList.innerHTML !== ""){
-            errorList.hidden = false;
-            event.preventDefault()
-        }
-    })
-}
\ No newline at end of file
+if (newLibraryForm) {
+  newLibraryForm.addEventListener("submit", (event) => {
+    let name = document.getElementById("libraryName");
+    let lat = document.getElementById("lat");
+    let lng = document.getElementById("lng");
+    let image = document.getElementById("library-image");
+    let fullnessVal = document.getElementById("fullness").value;
+    removeElementsByClass("error-list");
+    errorList.innerHTML = "";
+    errorList.hidden = true;
+    try {
+      validationsForStrings("Library Name", name.value, false, {
+        min: 3,
+        max: 40,
+      });
+      document.getElementById("nameError").innerText = "";
+    } catch (e) {
+      if (typeof e === "string" && e.startsWith("VError")) {
+        e = e.substr(1);
```

Comment:
What is changed here ?

Reply:
I mean the entire file

Reply:
Julien uses the prettier formatter and that's what these changes are